test/functional/helpers.lua

```diff
@@ -49,6 +53,10 @@ end
 
 local session, loop_running, loop_stopped, last_error
 
+local function set_session(s)
+  session = s
+end
```

Comment:
> After this PR I plan to refactor helpers.lua into a module with a new() constructor so that a test can control multiple sessions. Does that sound ok?

:+1: 

> Another thing that would be useful (for testing swapfiles) is ability to kill -9 the session. Any nice way to do that? I see that loop.c in the lua client does not send signals.

I added session:exit() to accept arguments for sending SIGTERM/SIGKILL: https://github.com/neovim/lua-client/blob/master/nvim/loop.c#L154-L156, but forgot to forward parameters from the session layer to the loop layer: https://github.com/neovim/lua-client/blob/master/nvim/session.lua#L127-L129 . It should be very simple to enable this functionality, could you send a PR to lua client?